docs/source/docker_setup.rst

```diff
@@ -0,0 +1,25 @@
+Docker Setup
```

Comment:
Does this page appear in the table of contents?

Reply:
Fixed.